manager/src/main/java/org/apache/hertzbeat/manager/scheduler/netty/process/HeartbeatProcessor.java

```diff
@@ -38,9 +38,14 @@ public HeartbeatProcessor(final ManageServer manageServer) {
     @Override
     public ClusterMsg.Message handle(ChannelHandlerContext ctx, ClusterMsg.Message message) {
         String identity = message.getIdentity();
-        boolean isChannelExist = this.manageServer.isChannelExist(identity);
-        if (!isChannelExist) {
-            log.info("the collector {} is not online.", identity);
+        boolean isChannelActive = this.manageServer.isChannelActive(identity);
+        if (!isChannelActive) {
+            this.manageServer.addChannel(identity, ctx.channel());
```

Comment:
if the collector is offline and re online throgh heartbeatProcessor, does we need add `this.manageServer.getCollectorAndJobScheduler().collectorGoOnline(collector, collectorInfo);` to re-dispath current collect jobs.